### PR TITLE
toast/item/loot/misc names

### PR DESCRIPTION
**Pull request by asiekierka**

### CODE REVIEW

**Review by Gegy — APPROVED** [review]

**Review by se7kn8 — APPROVED** [review]

File: mappings/net/minecraft/client/render/item/ItemRenderer.mapping [hunk 1]
```diff
@@ -35,19 +36,25 @@ CLASS dkm net/minecraft/client/render/item/ItemRenderer
...
 	METHOD a renderQuadColored (Lcpi;Ldft;I)V
+		ARG 3 color
 	METHOD a renderModelColored (Lcpi;Ljava/util/List;ILaxd;)V
-	METHOD a renderItemOverlaysInGUI (Lcqm;Laxd;II)V
+	METHOD a renderGuiItemOverlay (Lcqm;Laxd;II)V
 		ARG 1 fontRenderer
 		ARG 2 stack
 		ARG 3 x
-	METHOD a renderItemOverlaysInGUIWithText (Lcqm;Laxd;IILjava/lang/String;)V
+	METHOD a renderGuiitemOverlay (Lcqm;Laxd;IILjava/lang/String;)V
```
Comment (shedaniel) [inline mappings/net/minecraft/client/render/item/ItemRenderer.mapping]: Should It be `renderGuiItemOverlay`?

**Comment by asiekierka** [comment]
Could you have another look, please?

File: mappings/net/minecraft/client/render/item/ItemRenderer.mapping [hunk 2]
```diff
@@ -2,52 +2,73 @@ CLASS dkm net/minecraft/client/render/item/ItemRenderer
...
+	METHOD a renderGuiItem (Lahl;Laxd;II)V
 		ARG 2 itemStack
-		ARG 3 x
+		ARG 3 xPosition
+		ARG 4 yPosition
 	METHOD a hasDepthInGui (Laxd;)Z
-	METHOD a renderItemWithPropertyOverrides (Laxd;II)V
+	METHOD a renderGuiItemIcon (Laxd;II)V
 		ARG 1 stack
 		ARG 2 xPosition
```
Comment (Gegy) [inline mappings/net/minecraft/client/render/item/ItemRenderer.mapping]: `xPosition` and just `x` seem to be inconsistently used in the parameters here

Comment (modmuss50) [reply mappings/net/minecraft/client/render/item/ItemRenderer.mapping]: Yeah, check the rest of the class to ensure they all make sense. `renderGuiItemModel` uses `x` and `y` for example

File: mappings/net/minecraft/client/render/item/ItemRenderer.mapping [hunk 3]
```diff
@@ -2,52 +2,73 @@ CLASS dkm net/minecraft/client/render/item/ItemRenderer
...
+	METHOD a renderModelWithTint (Ldpw;I)V
+	METHOD a renderModel (Ldpw;ILaxd;)V
 	METHOD a renderItemModel (Ldpw;Laxd;)V
-	METHOD b renderItemAndGlowInGui (Laxd;II)V
+	METHOD a getOverriddenModel (Ldpw;Laxd;Lbcl;Lahl;)Ldpw;
+	METHOD b getModel (Laxd;)Ldpw;
+	METHOD b renderGuiItem (Laxd;II)V
 		ARG 1 stack
 		ARG 2 xPosition
+		ARG 3 yPosition
```
Comment (modmuss50) [inline mappings/net/minecraft/client/render/item/ItemRenderer.mapping]: Looking here it may not be your fault, as ARG 2 was already named `xPosition`

**Review by modmuss50 — CHANGES_REQUESTED** [review]
Almost there, just a small thing that @gegy1000 picked up on, not sure its entirely your fault as it appears to be already mixed in this class.

**Comment by asiekierka** [comment]
Updated.

**Review by Gegy — APPROVED** [review]

**Pull request merged**